src/test/java/org/embl/mobie/lib/create/ImagesCreatorTest.java

```diff
@@ -195,7 +195,7 @@ Object copyImageIntoDataset(boolean is2D ) {
         // NB: Avoid recursive class loading of ij.* classes.
         ImagePlus image = (ImagePlus) writeImageOutsideProject( is2D );
 
```

Comment:
Ideally, it would be great to add new tests to cover linking / copying etc for remote images. It should make the project creator easier to maintain - although, with the tests github action currently failing it's hard to spot when changes are breaking these tests.

Reply:
It would be great to add some tests!
Could you?
At least locally the tests are running and thus useful.